pkg/utils/net-attach-def_test.go

```diff
@@ -337,6 +337,48 @@ var _ = Describe("Netwok Attachment Definition manipulations", func() {
 		})
 	})
 
+	Context("create network statuses for a single interface which omits the sandbox info", func() {
+		var cniResult *cni100.Result
+
+		BeforeEach(func() {
+			cniResult = &cni100.Result{
+				CNIVersion: "1.1.0",
+				Interfaces: []*cni100.Interface{
+					{
+						Name: "foo",
+					},
+				},
+				IPs: []*cni100.IPConfig{
+					{
+						Address: *EnsureCIDR("10.244.196.152/32"),
+					},
+					{
+						Address: *EnsureCIDR("fd10:244::c497/128"),
+					},
+				},
+			}
+		})
+
+		It("creates network statuses with a single entry", func() {
+			networkStatuses, err := CreateNetworkStatuses(cniResult, "test-default-net-without-sandbox", true, nil)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(networkStatuses).To(WithTransform(
```

Comment:
we must clear the DNS information since we are using the [convertDNS](https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/blob/ed6baad2ba6958edf56e8437d78ad8fe9609738c/pkg/utils/net-attach-def.go#L37) function within `CreateNetworkStatuses` ...